numpy/core/src/multiarray/getset.c

```diff
@@ -938,6 +938,33 @@ array_matrix_transpose_get(PyArrayObject *self, void *NPY_UNUSED(ignored))
     return PyArray_MatrixTranspose(self);
 }
 
+static PyObject *
+array_ptp(PyArrayObject *self, void *NPY_UNUSED(ignored))
+{
+    PyErr_SetString(PyExc_AttributeError,
+                    "`ptp` has been removed from ndarray class. "
```

Comment:
Minor nit, here and in the other messages I'd say "was removed from the ndarray class in NumPy 2.0".

Reply:
Sure! I updated them.